static/app/components/sidebar/sidebarDropdown/switchOrganization.tsx

```diff
@@ -167,8 +167,9 @@ const SwitchOrganizationMenuActor = styled('span')`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  margin: 0 -${p => p.theme.sidebar.menuSpacing};
-  padding: 0 ${p => p.theme.sidebar.menuSpacing};
+  /* @TODO(jonasbadalic): the 15px is non standard spacing. Should it be space(2) which is 16px? */
+  margin: 0 -15px;
+  padding: 0 15px;
```

Comment:
since we’re using the `15px` in multiple places, I would suggest to either go with `space(2)` (I doubt the 1px will have a visible impact) or extract it to a constant

Reply:
I think so too, let me follow up with this and get a blessing from a designer :)